### PR TITLE
Jared/offseason

### LINKED ISSUE #35: Hanger Module Organization
This is a good start! Here's some suggestions

For the HangerModule class, can you implement the following:

- [x] MoveUp() -> Moves hanger module up
- [x] MoveDown() -> Moves hanger module down
- [x] Implement the initSendable(Builder) function for better modularity in implementing the ShuffleBoard elements

For the Hanger class, do the following
- [x] In HangStop() function, call the HangerModule.HangStop() functions instead of Move(0)
- [x] Implement GetLeftModule(), GetRightModule() to return access to the left and right hanger submoudles
- [x] Remove the (Left/Right)Hang(Up/Down/Stop/IsDown) commands and replace any bindings with Hanger.Get(Left/Right)Module().<function to use>. This is a neater way to do this.
- [x] Change the Hanger.initSendable to use the HangerModule.initSenable with the Left/Right submodules named.
- [x] Change Hanger Speed variables to be `static final`

### PR DESCRIPTION
**Pull request by ghost**
>Here is the pull Request.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -20,9 +20,9 @@ public class Robot extends TimedRobot {
   public void robotInit() {
     m_robotContainer = new RobotContainer();
     m_robotContainer.GetDriveTrainSub().ZeroGyro().schedule();
-    var cam = CameraServer.startAutomaticCapture();
-    cam.setResolution(100, 100);
-    cam.setFPS(60);
+    CameraServer.startAutomaticCapture();
+    // cam.setResolution(100, 100);
```
Comment (dlezcan1) [inline src/main/java/frc/robot/Robot.java]: @jforchheimer25 Remove the comments here entirely.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
-
-import frc.robot.commands.*;
-import frc.robot.subsystems.*;
+import frc.robot.commands.AutoIntake;
+import frc.robot.commands.SwerveDriveCommand;
+import frc.robot.subsystems.DriveTrain;
+import frc.robot.subsystems.Hanger;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.Shooter;
```
Comment (dlezcan1) [inline src/main/java/frc/robot/RobotContainer.java]: @jforchheimer25 Why did you alter the imports here?

File: src/main/java/frc/robot/subsystems/Hanger.java [hunk 3]
```diff
@@ -1,123 +1,87 @@
...
-
-  private RelativeEncoder hangerLeftMtrEnc = hangerLeftMtr.getEncoder();
-  private RelativeEncoder hangerRightMtrEnc = hangerRightMtr.getEncoder();
-
-  double hangSpeedUp = 0.75;
-  double hangSpeedDown = -0.75;
+  private HangerModule m_leftHanger;
+  private HangerModule m_rightHanger;
+
+  public static final double hangSpeedUp = 0.75;
```
Comment (dlezcan1) [inline src/main/java/frc/robot/subsystems/Hanger.java]: @jforchheimer25 add the `s_` to the names to indicate these are static variables

**Review by dlezcan1 — CHANGES_REQUESTED** [review]

**Comment by dlezcan1** [comment]
@jforchheimer25 Any updates?

**Comment by ghost** [comment]
> @jforchheimer25 Any updates?

Need to Test on bot. I have no reason to see why this wouldnt work though.

**Comment by dlezcan1** [comment]
Please address the review comments so that we can merge the PR

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (ghost) [reply src/main/java/frc/robot/RobotContainer.java]: Ill revert it

**Pull request merged**